watchdog-agent/src/main/java/io/github/algomaster99/Options.java

```diff
@@ -35,7 +43,7 @@ public class Options {
     private Path externalJars;
 
     public Options(String agentArgs) {
-        String[] args = agentArgs.split(",");
+        String[] args = agentArgs.split(",", -1);
```

Comment:
Purpose?

Reply:
Was a reflex to add this as you normally always want to split with `-1` in java,if you even you split. The -1 retains empty parts of the split `a,b,c,` => ["a","b","c",""]. Will remove it.